### DISCUSSION TITLE
Segítség kérés

### DISCUSSION THREAD

**Discussion by boyocska65** [amaximus/met_alerts_hu#13] [Q&A]
Odáig eljutottam, hogy a HACS-ban hozzáadtam a repo-t, de nem tiszta, mit írjak a sensors.yaml ill. configuration.yaml fájlokba, hogy működjön. Most kezdtem a HA-t használni, de abszolút kezdőként kicsit szájbarágşabb útmutatót szeretnék. Légyszíves segíts.

**Comment by amaximus** [comment]
A sensors.yaml-ba - ha külön van - akkor a következőket kellene írni:
```
- platform: met_alerts_hu
  name: 'MET alerts'
  region_id: 101 # Budapest
  county_id: 13 # Pest county
```
A region_id a járásnak a száma [erről ](https://www.met.hu/idojaras/veszelyjelzes/index.php)a térképről, a county_id meg a megye sorszáma ha a megyék ábécé sorrendbe vannak rendezve (2: Baranya; ...; 20: Zala).

A configuration.yamlüba nem kell semmi, feltéve, ha sensors.yamlüra való hivatkozás helyes.
Természetesen a sensors.yaml módosítása után a HA-t újra kell indítani.

**Reply by boyocska65** [reply]
Elvileg most megjelent a sensor, az egyéni kártyára betettem az oldalon talált kódot és várom, hogy legyen riasztás, mert így csak azt látom, hogy szintaktikailag nincs provléma. Köszönöm!

**Comment by amaximus** [comment]
Nincs mit.
